src/objects/list.rs

```diff
@@ -76,20 +80,29 @@ impl PyList {
         assert!(r == 0);
     }
 
+    /// Appends an item to the end of the list
+    pub fn append_item(&self, _py: Python, item: PyObject) {
```

Comment:
why not call it just `append`?  It would be consistent with the Python namings, and there's no ambiguity about what it does.